app/controllers/HomeController.scala

```diff
@@ -21,7 +21,7 @@ class HomeController @Inject()(requestService: RequestService, cc: ControllerCom
 
   def parallelRequests() = Action.async {
     val futures = (1 to 10).map( _ =>
-      requestService.makeRequest(s"http://52.42.198.2:8080?p=${rand.nextInt(99999) + 1}"),
+      requestService.makeRequest(s"http://controller/?p=${rand.nextInt(99999) + 1}"),
```

Comment:
Hi @wsargent , thanks for paying attention to this benchmark. I wanted to hit remote service to make the benchmark more realistic. Btw, what's the `http://controller`?

Reply:
That is a hostname that I can control the IP address of.  I can change it so it’s pupled from config.